recipes/cuda-cudart/meta.yaml

```diff
@@ -0,0 +1,145 @@
+{% set name = "cuda-cudart" %}
+{% set version = "12.0.107" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_cudart/{{ platform }}/cuda_cudart-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: c0e16e9593a69afe4997ae96a8b024e751ebce0254523fd969b08bb028582d23  # [linux64]
+  sha256: d785fd613e35e3761fdb0016a31a56417f570605cbfb9a93ca9dc68be6789188  # [ppc64le]
+  sha256: 5de248c6c203f6c91ce34256bbfc028bc87c1b0c552f9f51edb65e60ad665f63  # [aarch64]
+  sha256: 1cd8516feee068c6d718453e76b0dfcbee27844a35d4f3aa608a3d316190309c  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+  run_exports:                # [linux]
+    strong:                   # [linux]
+      - __glibc >=2.17        # [linux]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - sysroot_{{ target_platform }} 2.17   # [linux]
+    - __glibc >=2.17                       # [linux]
+  run:
+    - sysroot_{{ target_platform }} 2.17   # [linux]
+    - __glibc >=2.17                       # [linux]
+
+outputs:
+  - name: cuda-cudart
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcudart.so.{{ version }}     # [linux]
+        - if not exist %LIBRARY_BIN%\cudart64_*.dll exit 1   # [win]
+    files:
+      - lib/*.so.*                # [linux]
```

Comment:
My previous concern seems valid. This wildcard is too wild.

Reply:
(ps: I am not 100% certain if this is the line to blame...)

Reply:
Yes, you need `files:` and `script: install.sh` so that the copying happens in the output and not in the top-level `build.sh`. `install.sh` can just be the same as `build.sh`.

Reply:
This should be fixed now, I don't see extra files in any of the outputs now. 
I wasn't having much luck with the `install.sh` suggestion, the extra files were still being picked up. We can revisit this if needed.

Reply:
With regards to this, I tried this in [here](https://github.com/conda-forge/staged-recipes/pull/21902/files#diff-027adc5f890b1eae5cf8f74500eb61472844cfb38116b05297672036754b1e02R30) (probably incorrect?).  I.e. adding `script:` with the script being basically the current one with some tweaks) and it didn't work, the windows core libraries are still all showing up (the other ones are probably specific enough anyway in that recipe).

So I am wondering if narrow specific pattern may be the easier solution after all?

Reply:
Yeah that matches what I was seeing here, and ultimately went with the more specific glob. This should be fine in the runtime package as long as we double check the source contents.

Reply:
>  I wasn't having much luck with the install.sh suggestion, the extra files were still being picked up. We can revisit this if needed.

Did you keep `build.sh`? You have to delete that.

Reply:
Yes, I suspect the point is that the `install.sh` when done like this should do all the work and we would remove `files:`?

Reply:
Yes, that's one use-case, but also you can have one `install.sh` for multiple outputs and you can use `files:` to pattern match certain files for each output.

Reply:
Hmm, I tried a few things and the pattern didn't work?  Putting it top-level was rejected by the linter, putting it in `build: script:` seemed to not work/do something different?

Reply:
Did you have a recent conda-build? Having both `script` and `files` is a new feature

Reply:
Hmmm, I tried on CI?  Unless I made a weird error (which is entirely possible).

Reply:
https://github.com/conda-forge/staged-recipes/pull/21902#issuecomment-1423870575 is with it on the top level.  But maybe I am misunderstanding?